Source/WebCore/editing/markup.cpp

```diff
@@ -1452,9 +1449,9 @@ ExceptionOr<Ref<DocumentFragment>> createContextualFragment(Element& element, co
     return fragment;
 }
 
-static inline bool hasOneTextChild(ContainerNode& node)
+static inline RefPtr<Text> dynamicOnlyTextChild(ContainerNode& node)
```

Comment:
I don’t think “dynamic only text child” expresses “return a single child if it’s text”. I would call it `singleTextChild`, which isn’t perfect either but I think is better.